domain/src/main/java/com/softserve/borda/entities/User.java

```diff
@@ -39,8 +39,6 @@
     private String lastName;
 
     private boolean enabled = true;
-
-    @Column(name = "avatar")
```

Comment:
Why did you remove mapping?

Reply:
Not sure. Must have been accidental change, my bad. Brought it back